apiclient/types/assitant.go

```diff
@@ -5,6 +5,7 @@ type Assistant struct {
 	Name        string     `json:"name"`
 	Description string     `json:"description"`
 	Icons       AgentIcons `json:"icons"`
+	EntityId    string     `json:"entityId"`
```

Comment:
This is the Go standard.
```suggestion
	EntityID    string     `json:"entityID"`
```